src/components/Input/DropDown/DropDownGroup.styles.js

```diff
@@ -44,7 +44,7 @@ export const StyledGroup = styled.label`
     border: solid 1px ${getThemeValue("gray02")};
     border-radius: ${small} ${small} 0 0;
     /* for the purpose of the correct box-shadow */
-    z-index: ${zIndex.layout.overlay + 1};
+    z-index: ${zIndex.default};
```

Comment:
I consider this a breaking design change. Not sure we can make this adjustment at this point.

I'd prefer to have devs override the `z-index`, as opposed to changing the base values.

Reply:
Agree, fixed it in next-renderer.

Reply:
Thank you!

Reply:
I think this fix in `src/components/Button/Base.js` should be merged anyway. I'll edit the PR.

Reply:
👍